kratos/python/add_matrix_to_python.cpp

```diff
@@ -89,6 +90,33 @@ namespace Python
 
           return matrix;
         }));
+
+        matrix_binder.def(py::init([](const py::list& input){
+          std::size_t num_rows = input.size();
+          std::size_t num_cols = 0;
+
+          //To calculate the number of columns
+          for(std::size_t i = 0 ; i< input.size() ; i++)
+          {
+            auto v_row = Vector(py::cast<Vector>(input[i]));
+            if(num_cols < v_row.size())
+              num_cols = v_row.size();
+          }
```

Comment:
Can't we directly take `input[0].size()` to avoid this? (related to my comment below)

Reply:
cant do `input[0].size()` directly. 
But yes, I can access the number of columns like 
`std::size_t num_cols = (Vector(py::cast<Vector>(input[0]))).size();
`